Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -84,6 +84,31 @@ public enum NIORenegotiationSupport {
     case always
 }
 
+/// Signature algorithms. The values are defined as in TLS 1.3
+public struct SignatureAlgorithm : RawRepresentable {
```

Comment:
While we're here let's also add `Hashable` to this, it'll make our lives a little easier.